app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -32,8 +55,34 @@ fun PlotSurface() {
         //  This should include a Column composable that
         //  includes a Map, and two MapSlider composables
         //  (one slider for each axis).
+        Column(
```

Comment:
Feel free to remove the TODO comments here and for the Map composable as well.  The TODOs are complete since you filled in the code.

Reply:
Removed! thanks